code/modules/organs/organ_internal.dm

```diff
@@ -208,10 +208,27 @@ obj/item/organ/vaurca/neuralsocket/process()
 	parent_organ = "chest"
 	icon = 'icons/mob/alien.dmi'
 	icon_state = "breathing_app"
-	robotic = 2
+	robotic = 0
 
 obj/item/organ/vaurca/breathingapparatus/process()
 	return
 
 /obj/item/organ/vaurca/breathingapparatus/removed()
 	return
+
+/obj/item/organ/vaurca/filtrationbit
+	name = "filtration bit"
+	organ_tag = "filtration bit"
+	parent_organ = "head"
+	icon = 'icons/mob/alien.dmi'
+	icon_state = "filter"
+	robotic = 2
+
+/obj/item/organ/vaurca/filtrationbit/process(var/mob/living/carbon/human/owner)
```

Comment:
Unnecessary.

If you write a proc that simply goes `..()`, then you don't generally need to write it at all.